pkg/logs/logs.go

```diff
@@ -58,19 +61,45 @@ func (writer GlogWriter) Write(data []byte) (n int, err error) {
 }
 
 // InitLogs initializes logs the way we want for kubernetes.
-func InitLogs(fs *flag.FlagSet) {
+func InitLogs() {
 	logs.InitLogs()
 
-	if fs == nil {
-		fs = flag.CommandLine
-	}
-	initDeprecatedFlags(fs)
-	_ = fs.Set("logtostderr", "true")
-
 	log.SetOutput(GlogWriter{})
 	log.SetFlags(0)
 }
 
+func AddFlags(opts *logs.Options, fs *pflag.FlagSet) {
+	// init deprecated flags
```

Comment:
I don't think "deprecated flags" means something in the context of this PR.

Do you mean that what we are doing here is cherry-picking the flags provided by `k8s.io/component-base` because all the flags that `k8s.io/component-base` provides aren't yet supported by cert-manager, for example "--log-json-info-buffer-size"? In that case, I'd suggest rephrasing the comment.

Reply:
I'll just remove this comment.